### ISSUE TITLE
docker image elasticsearch/elasticsearch:5.4.2 not found

### ISSUE THREAD

**Issue by samuelcolvin** [elastic/elasticsearch#25027]
Straight from your docs:

> The Docker image can be retrieved with the following command:
> `docker pull docker.elastic.co/elasticsearch/elasticsearch:5.4.2`

Which gives:

```
➤  docker pull docker.elastic.co/elasticsearch/elasticsearch:5.4.2
Pulling repository docker.elastic.co/elasticsearch/elasticsearch
Error: image elasticsearch/elasticsearch:5.4.2 not found
```

`5.4.1` works but not `5.4.2`.

Am I going mad or does one of your 4 main ways of installing elastic search currently not work?

**Comment by jasontedor** [comment]
5.4.2 is not released.

**Comment by samuelcolvin** [comment]
https://www.elastic.co/guide/en/elasticsearch/reference/current/docker.html

5.4.2 is referenced 6 times.

**Comment by jasontedor** [comment]
The version constant was bumped prematurely, it's fixed in 6f88b5b92be107b92b95bfd9ddd0fa9dc0e79a34. The page will regenerate soon.
